backoffice/templates/backoffice/home.html

```diff
@@ -3,6 +3,24 @@
 {% block heading %}Dashboard{% endblock %}
 
 {% block main %}
+  {% if drivers_pending > 0 or accounts_pending > 0 %}
```

Comment:
Are `drivers_pending` and `accounts_pending` already available in the context for this view, or is there a bit missing from this commit that adds them in the view class?

Reply:
This commit got lost somewhere, have rewritten it now.